src/Api/Controllers/AccountsController.cs

```diff
@@ -479,9 +511,9 @@ public async Task Delete([FromBody]DeleteAccountRequestModel model)
                 throw new UnauthorizedAccessException();
             }
 
-            if (!await _userService.CheckPasswordAsync(user, model.MasterPasswordHash))
+            if (!await _userService.VerifyPasswordOrOTPAsync(user, model.Secret))
             {
-                ModelState.AddModelError("MasterPasswordHash", "Invalid password.");
+                ModelState.AddModelError(string.Empty, "User verification failed.");
```

Comment:
Why string.Empty for the key here instead of 'Secret'? There are a couple instances of this, I'm going to make a brief note on each one.

Reply:
We usually use the explicit field, since it could be either `MasterPasswordHash` or `OTP` we omit it and give a generic error.

Reply:
Secret is an explicit field of the request model, I would say that is still valuable information for these messages.

Reply:
@addisonbeck Secret is an explicit field on the backend, but this data is send to the clients, where secret doesn't exist.